packages/build-tools/src/steps/easFunctionGroups.ts

```diff
@@ -4,12 +4,15 @@ import { CustomBuildContext } from '../customBuildContext';
 
 import { createEasBuildBuildFunctionGroup } from './functionGroups/build';
 import { createEasMaestroTestFunctionGroup } from './functionGroups/maestroTest';
+import { createEasSubmitBuildFunctionGroup } from './functionGroups/submit';
 
 export function getEasFunctionGroups(ctx: CustomBuildContext): BuildFunctionGroup[] {
   const functionGroups = [createEasMaestroTestFunctionGroup(ctx)];
 
   if (ctx.hasBuildJob()) {
-    functionGroups.push(...[createEasBuildBuildFunctionGroup(ctx)]);
+    functionGroups.push(
+      ...[createEasBuildBuildFunctionGroup(ctx), createEasSubmitBuildFunctionGroup(ctx)]
```

Comment:
We should make it so `eas/submit` does not require a build job to run. It currently runs in a generic job and it should remain like this.

When this lands we'll replace implementations in `WorkflowSubmitJob` with calls to `eas/submit`.